llama-index-core/llama_index/core/vector_stores/simple.py

```diff
@@ -62,8 +62,17 @@ def filter_fn(node_id: str) -> bool:
             if metadata_value is None:
                 filter_matches = False
             elif isinstance(metadata_value, list):
-                if filter_.value not in metadata_value:
-                    filter_matches = False
+                match filter_.operator:
```

Comment:
this syntax is only supported in python3.10+ I think

We need to support python3.8+

Reply:
Updated to remove match